### PR TITLE
Relax II's CSP-policies, to enable calling custom domains

### PR DESCRIPTION
**Pull request by przydatek**
II's CSP policies were too restrictive to support custom domains.  This PR relaxes the policies.

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

File: src/internet_identity/src/http.rs [hunk 1]
```diff
@@ -181,11 +181,11 @@ fn content_security_policy_header(integrity_hashes: Vec<String>) -> String {
         )
     };
 
-    let connect_src = "'self' https://identity.internetcomputer.org https://icp-api.io https://*.icp0.io https://*.ic0.app";
+    let connect_src = "'self' https:";
 
     // Allow connecting to localhost, including subdomains, on http and https for local development purposes
     #[cfg(feature = "dev_csp")]
-        let connect_src = format!("{connect_src} http://localhost:* http://*.localhost:* https://localhost:* https://*.localhost:*");
+    let connect_src = format!("{connect_src} http://localhost:* http://*.localhost:*");
```
Comment (frederikrothenberger) [inline src/internet_identity/src/http.rs:188]:
```suggestion
    let connect_src = format!("{connect_src} http:");
```

Comment (frederikrothenberger) [reply src/internet_identity/src/http.rs:188]: This way, in dev mode connections to insecure resources is allowed. Please also adjust the comment above.

**Review by frederikrothenberger — APPROVED** [review]
Thanks, LGTM!
Please do a quick search throughout the project for `DEV_CSP` and adjust the documentation.

File: src/internet_identity/src/http.rs (hunk 1 above)
Comment (przydatek) [reply src/internet_identity/src/http.rs]: Done.

**Pull request merged**